resources/metadata/apiKeyValueMapUpload.yaml

```diff
@@ -0,0 +1,49 @@
+metadata:
+  name: apiKeyValueMapUpload
+  description: Upload an api key value map artifact in to the API Portal
```

Comment:
does the upload create and/or update a key value map?

Reply:
just create operation